splash/lua.py

```diff
@@ -203,7 +208,7 @@ def python2lua(lua, obj, max_depth=100):
     if isinstance(obj, dict):
         return lua.table_from({
             python2lua(lua, key, max_depth-1): python2lua(lua, value, max_depth-1)
-            for key, value in obj.iteritems()
+            for key, value in list(obj.items())
```

Comment:
One option is six.iteritems, another is just `obj.items()`. `obj.items` will be a bit less efficient in Python 2.x. `list(obj.items())` looks pointless.